docs-build/docusaurus.config.js

```diff
@@ -0,0 +1,123 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require('prism-react-renderer/themes/github')
+const darkCodeTheme = require('prism-react-renderer/themes/dracula')
+// baseUrl: the name of the repository
+// url: after your githup name .github.io
+// github user or organization that own the repository
+// projectName: the name of the githuo repository
+// please see the link: https://docusaurus.io/docs/deployment
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'SLIC Starter',
+  tagline: 'SLIC Starter is a complete starter project for production-grade serverless applications on AWS.',
+  url: 'https://direnakkocdemir.github.io',
```

Comment:
URL and repo references in this file should be changed to the main slic-starter repo